plugins/module_utils/user.py

```diff
@@ -725,7 +725,7 @@ def privileges_grant(cursor, user, host, db_table, priv, tls_requires, maria_rol
     try:
         cursor.execute(query, params)
     except (mysql_driver.ProgrammingError, mysql_driver.OperationalError, mysql_driver.InternalError) as e:
-        raise InvalidPrivsError("Error granting privileges, invalid priv string: %s" % priv_string)
+        raise InvalidPrivsError("Error granting privileges, invalid priv string: %s , query: %s , exception: %s." % (priv_string, query, str(e)))
```

Comment:
```suggestion
        raise InvalidPrivsError("Error granting privileges, invalid priv string: %s , query: %s , params: %s , exception: %s." % (priv_string, query, params, str(e)))
```
maybe?

Reply:
I think the params are visible if you increase the logging in the normal ansible output ie with -vvv, but I can include it indeed if you want it repeated

Reply:
@bizmate i believe it's better to include them as the params that are passed to cursor.execute can differ from those we pass, i.e. they can be changed during execution

Reply:
Added, marking as resolved